### DISCUSSION TITLE
Action打包成功但是没有Artifacts

### DISCUSSION THREAD

**Discussion by sc03209** [tw93/Pake#506] [Q&A]
![image](https://github.com/tw93/Pake/assets/44691266/2523c265-e876-4ef6-befc-cf361d22679c)
小白一枚，按照教程填好了之后打包显示成功，但是不出现安装包，一直是这个报错，请问各位大佬是什么原因啊？

**Comment by tw93** [comment]
<img width="1104" alt="image" src="https://github.com/tw93/Pake/assets/8736212/6c6f5fd6-0702-40c5-ae0a-b5c653f5b5e4">
ico 地址不对

**Reply by sc03209** [reply]
![image](https://github.com/tw93/Pake/assets/44691266/69e0878a-6b89-4a5e-bead-15ac9ff6a976)
![image](https://github.com/tw93/Pake/assets/44691266/c08b4291-adfb-4237-a874-a187c1850469)

更改了地址之后还是显示错误
